coding_systems/snomedct/coding_system.py

```diff
@@ -25,9 +27,11 @@ def lookup_names(codes):
 
 def search(term):
     return set(
-        Concept.objects.filter(
-            descriptions__term__contains=term, descriptions__active=True, active=True
-        ).values_list("id", flat=True)
+        Concept.objects.filter(active=True)
+        .filter(
+            Q(descriptions__term__contains=term, descriptions__active=True) | Q(id=term)
+        )
```

Comment:
Are these in two different `.filter()` calls on purpose?

Reply:
Yes, I find `.filter(q1).filter(q2 | q3)` slightly easier to parse than `.filter(q1 & (q2 | q3))`.

Reply:
+1 to that!